packages/fx-core/src/plugins/resource/simpleauth/constants.ts

```diff
@@ -94,6 +94,24 @@ export class Constants {
     static readonly FreeServerFarmsQuotaErrorHelpLink = "https://aka.ms/teamsfx-sa-help#freeserverfarmsquotaerror";
 }
 
+export class Telemetry {
+    static component = "component";
+    static errorCode = "error-code";
+    static errorType = "error-type";
+    static errorMessage = "error-message";
+    static userError = "user";
+    static systemError = "system";
+  }
+  
+  export class Stage {
```

Comment:
Are these stage used as event names for error? If so seems not same as the ones defined in https://github.com/OfficeDev/TeamsFx/blob/8f8c5ea16afadb40f21a9c4df32146fd15e37701/packages/fx-core/src/plugins/resource/aad/constants.ts#L95 Besides, please check with Changyong about the names. From the document I didn't see local-debug. However there are f5-start, f5